### PR TITLE
adds more info to flash messages for creating account

### PR DESCRIPTION
**Pull request by alfosco**
This adds more info to flash messages for creating an account.
-adds partial to views/users
-adds some content to views/users/new.html.erb
-also changes fields for password and password confirmation from text_fields to password_fields

flash messages

### CODE REVIEW

**Comment by cmacaulay** [comment]
@s-espinosa @AliSchlereth is that what you were envisioning? Or were you thinking of warnings as you fill out the form? Thanks for letting us know!

File: app/views/users/_error_messages.html.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<% if object.errors.any? %>
```
Comment (cmacaulay) [inline app/views/users/_error_messages.html.erb]: this is sweet @alfosco!

File: app/views/users/new.html.erb [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
   <div class="row center">
     <%= form_for @user, class: "col s12" do |f| %>
+    <%= render 'users/error_messages', object: f.object %>
```
Comment (cmacaulay) [inline app/views/users/new.html.erb]: 👏 👏

Comment (s-espinosa) [inline app/views/users/new.html.erb]: @cmacaulay This is slightly different from the pattern I've seen before (in the past have seen the errors been captured in the controller, and then iterate over them in the view), but I think this pretty much does what you need it to do.

One thing: recommend not displaying this if there are no errors (maybe I'm missing it, but it seems like it's always displaying whether or not there are any errors).

**Pull request merged**